src/styles/_catalog.scss

```
@@ -14,6 +14,12 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition-duration: 300ms;
```

```suggestion
    transition-duration: 300ms;
    transition-property: transform;
```
Необходимо явно указывать свойства, которые будут анимированы, иначе используется transition: all
transition: all достаточно требователен к ресурсам браузера и на старых компьютерах может ухудшать пользовательский опыт

---

Здесь то же самое